lang/English/modules/settings.php

```diff
@@ -45,7 +45,7 @@
 define('feed_url', "Feed URL");
 define('feed_url_info', "GrayCube.com is sharing a LGSL feed on the URL:<br><b>http://www.greycube.co.uk/lgsl/feed/lgsl_files/lgsl_feed.php</b>");
 define('charset', "Character Encoding");
-define('charset_info', "UTF8, ISO, ASCII, etc... Left it blank to use ISO encoding.");
+define('charset_info', "UTF8, ISO, ASCII, etc... Leave it blank to use ISO encoding.");
```

Comment:
It seems English/globals.php file uses ISO as lang_charset, a quick look at some of the other language globals, and they use UTF-8.

It'd probably be good to change that to UTF-8 also, just for the sake of consistency. Afterwards, the translation above could be updated.